src/common/calendar/_timePickerCalendar.scss

```diff
@@ -32,72 +32,76 @@
     }
   }
 
-  @media (max-width: $screen-xs-max) {
-    .fc-time-grid .fc-slats td {
-      height: 2em !important;
-    }
-
-    .fc-view-container {
-      overflow: scroll;
+  .fc-header-toolbar {
+    .fc-left {
+      width: 30%;
+      .fc-button-group {
+        .fc-button {
+          flex-grow: 0;
+        }
 
-      .fc-timeGridWeek-view {
-        width: 700px;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
       }
     }
 
-    .fc-header-toolbar {
-      display: flex;
-
-      .fc-left,
-      .fc-right {
-        width: 30%
-      }
+    .fc-today-button {
+      border: 1px solid #000000;
+      border-radius: 0;
+      padding: 0.2em 0.3em;
+      color: #000000;
+      background: $white;
+      font-weight: $font-weight-bold;
 
 
-      .fc-button {
-        border: none;
-        border-radius: 0;
-        padding: 0.2em 0.3em;
+      &:disabled {
+        border:none;
         background: $hel-coat;
         font-weight: $font-weight-bold;
+        color: $white;
+        opacity: 1;
+      }
+    }
 
-        &:disabled {
-          border: 1px solid $black;
-          color: $black;
-          background: $white !important;
-        }
+    .fc-prev-button,
+    .fc-next-button {
+      margin: 0 !important;
+      background: $white;
+      color: #000000;
+      border:0;
+    }
 
-        &.fc-timeGridDay-button,
-        &.fc-timeGridWeek-button {
-          &.fc-button-active {
-            display: none;
-          }
-        }
-      }
+    .fc-timeGridWeek-button,
+    .fc-timeGridDay-button {
```

Comment:
Have you check out how its look like on every screen size ?

Reply:
Added one more media query for really small screens. Looks OK.